docs/CryptolPrims.md

```diff
@@ -53,6 +71,10 @@ Boolean
     (^)   : a -> a -> a
     (~)   : a -> a
 
+    (==>) : Bit -> Bit -> Bit
+    (/\)  : Bit -> Bit -> Bit
+    (\/)  : Bit -> Bit -> Bit
+
```

Comment:
Honestly, it is such a small change that I don't think it is really worth---it is only a change to docs after all. 

I'd say just merge it in.